### PR TITLE
Build restructuring and fixing the builds

### PR DESCRIPTION
**Pull request by tboerger**
Beside the title I have also added a .appveyor.yml to get the windows image built automatically.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,28 @@
 # drone-git
 
 [![Build Status](http://beta.drone.io/api/badges/drone-plugins/drone-git/status.svg)](http://beta.drone.io/drone-plugins/drone-git)
+[![Join the chat at https://gitter.im/drone/drone](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/drone/drone)
 [![Go Doc](https://godoc.org/github.com/drone-plugins/drone-git?status.svg)](http://godoc.org/github.com/drone-plugins/drone-git)
 [![Go Report](https://goreportcard.com/badge/github.com/drone-plugins/drone-git)](https://goreportcard.com/report/github.com/drone-plugins/drone-git)
-[![Join the chat at https://gitter.im/drone/drone](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/drone/drone)
+[![](https://images.microbadger.com/badges/image/plugins/git.svg)](https://microbadger.com/images/plugins/git "Get your own image badge on microbadger.com")
```
Comment (metalmatze) [inline README.md]: This only links the base image?

Comment (tboerger) [reply README.md]: What do you mean? It will display the plugins/git image size as a badge.

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,13 @@
-FROM alpine:3.5
+FROM plugins/base:amd64
+MAINTAINER Drone.IO Community <drone-dev@googlegroups.com>
```
Comment (mrueg) [inline Dockerfile]: Maintainer tag is deprecated, should be replaced with a label if you want one. https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

File: README.md [hunk 3]
```diff
@@ -1,37 +1,28 @@
 # drone-git
 
 [![Build Status](http://beta.drone.io/api/badges/drone-plugins/drone-git/status.svg)](http://beta.drone.io/drone-plugins/drone-git)
+[![Join the chat at https://gitter.im/drone/drone](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/drone/drone)
```
Comment (mrueg) [inline README.md]: Gitter isn't available anymore.

**Comment by tonglil** [comment]
@mrueg could you submit a PR please :)

**Pull request merged**